components/scream/tests/uncoupled/rrtmgp/rrtmgp_stand_alone.cpp

```diff
@@ -132,13 +134,25 @@ namespace scream {
         GasConcs gas_concs;
         read_atmos(inputfile, p_lay, t_lay, p_lev, t_lev, gas_concs, ncol);
 
+        // Create dummy values for sfc_alb_dir and sfc_alb_dif
+        // These are needed for compatibility with other rrtmgp tests
+        real2d sfc_alb_dir("sfc_alb_dir", ncol, nswbands);
+        real2d sfc_alb_dif("sfc_alb_dif", ncol, nswbands);
+
         // Setup dummy problem
         rrtmgpTest::dummy_atmos(
             inputfile, ncol, p_lay, t_lay,
             sfc_alb_dir, sfc_alb_dif, mu0,
             lwp, iwp, rel, rei, cld
         );
-        //
+
+        // Set surface albedos to 0.06. This is the initial condition used
+        // is other rrtmgp tests.
+        memset(sfc_alb_dir_vis , 0.06_wp );
+        memset(sfc_alb_dir_nir , 0.06_wp );
+        memset(sfc_alb_dif_vis , 0.06_wp );
+        memset(sfc_alb_dif_nir , 0.06_wp );
```

Comment:
@brhillman These few lines in rrtmgp_standalone (137-141 above as well) are the tradeoff for not changing any of the other rrtmgp tests.